lib_src/src/fixed_factor_of_3_voice/src_ff3v_fir.h

```diff
@@ -21,6 +21,13 @@ extern const unsigned src_ff3v_fir_comp_q_us;
 extern const int32_t src_ff3v_fir_comp_us;
 
 extern int32_t src_ff3v_fir_coefs_debug[SRC_FF3V_FIR_NUM_PHASES * SRC_FF3V_FIR_TAPS_PER_PHASE];
```

Comment:
The non-const debugging array has not been made usable from C.  Is that your intention?

Reply:
It is already usable from C. It seems that single dimensional arrays don't throw the same linker error that multi-dimensional arrays do when sharing them between C and XC.